docs/docs/DOCS_README.md

```diff
@@ -22,15 +22,17 @@ Install all dependencies required for building docs (mainly `mkdocs` and its ext
 
 - [Install poetry](https://python-poetry.org/docs/#installation) - this will help you manage package dependencies
```

Comment:
Is this whole page a duplicate of docs/DOCS_README?

Reply:
yes it gets copied into here automatically so that it can be included in the website lol